addons/l10n_in_ewaybill_stock/models/stock_picking.py

```diff
@@ -22,10 +22,6 @@ def action_l10n_in_ewaybill_create(self):
             raise UserError(_("Please set HSN code in below products: \n%s", '\n'.join(
                 [product.name for product in product_with_no_hsn]
             )))
-        if lines_with_no_tax := self.move_ids.filtered(lambda line: not line.ewaybill_tax_ids):
```

Comment:
@jco-odoo I forgot we were already supposed remove it let it be in this PR.
https://github.com/odoo/odoo/pull/169760